docs/.vitepress/config.ts

```diff
@@ -172,15 +175,18 @@ export default defineConfig({
       message: 'Released under the <a rel="license" href="https://creativecommons.org/licenses/by-sa/3.0/">CC BY-SA 3.0 Deed License</a>.',
       copyright: `Copyright © 2013 - ${new Date().getFullYear()}`
     },
-    markdown: {
-      theme: 'material-palenight',
-      lineNumbers: true,
-      config: md => {
-        require('vuepress-plugin-glossary').extendMarkdown(md);
-      },
-      toc: {
-        level: [2]
+  },
+  markdown: {
+    theme: 'material-theme-palenight',
+    lineNumbers: true,
+    config: md => {
+      md.use(markdownItDeflist);
+      for (const pattern of patterns) {
+        md.use(markdownItRegexp(pattern.regex, pattern.modifier));
```

Comment:
I'd rather we used the same method of extending the markdown from a plugin like before, i.e. define an `extendMarkdown` function in term-parser that we call here so that the config.ts doesn't need to know in detail what term-parser does.

Reply:
Yeah, probably best.